src/conf_mode/interfaces-wireguard.py

```diff
@@ -105,6 +107,9 @@ def verify(wireguard):
         if peer['public_key'] in public_keys:
             raise ConfigError(f'Duplicate public-key defined on peer "{tmp}"')
 
+        if 'disable' not in peer and is_wireguard_key_pair(wireguard['private_key'], peer['public_key']):
+            raise ConfigError(f'Peer "{tmp}" has the same public key as the router"')
```

Comment:
There's a stray double quote at the end of the string.

Reply:
done